http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/CookiesIT.java

```diff
@@ -52,6 +53,26 @@ void testSameSiteAttributeAddedByVertxHttpExt() {
                 .cookie("vertx", detailedCookie().sameSite(sameSite).secured(true));
     }
 
+    @Test
+    void testCookiesSerialization() {
+        given()
+                .get("/cookie/newcookie-serialization")
+                .then()
+                .statusCode(200)
+                .body(stringContainsInOrder(String.format("\"name\":\"%s\"", TEST_COOKIE),
+                        String.format("\"value\":\"%s\"", TEST_COOKIE_VALUE)));
+    }
+
+    @Test
+    void testCookiesSerialization3() {
```

Comment:
What happen to `testCookiesSerialization3` :-) This is unimportant, personally I think we don't need to have one call per one method, but if you prefer it this way, please find better name, because of skipping `2`. Maybe something that describes how they differ?

Reply:
I update it. The naming was when I tested few things and didn't change it to correct name.

But what exactly happen to `testCookiesSerialization2`, just cookie monster stole it while I push the code.